src/main/java/co/reachfive/identity/sdk/demo/AuthenticatedActivity.kt

```diff
@@ -49,10 +49,10 @@ class AuthenticatedActivity : AppCompatActivity() {
 
         devicesBinding = WebauthnDevicesBinding.inflate(layoutInflater)
 
-        this.authToken = intent.getParcelableExtra(AUTH_TOKEN)
+        this.authToken = intent.getParcelableExtra(AUTH_TOKEN)!!
```

Comment:
Not sure why I had a compilation error here, I couldn't find a diff regarding parcelable in Android SDK 29 -> 31.